Sources/Common/Core/Math/index.js

```diff
@@ -881,7 +881,7 @@ export function jacobiN(a, n, w, v) {
     tmp = w[k];
     for (i = j + 1; i < n; i++) {
       // boundary incorrect, shifted already
-      if (w[i] >= tmp) {
```

Comment:
such fix, if needed, must be done in a separate PR

Reply:
It has been reverted in further commits. This change does not appear on the final changes anymore.